vue/components/ui/molecules/bezier-curve/bezier-curve.vue

```diff
@@ -0,0 +1,377 @@
+<template>
+    <svg
+        class="bezier-curve"
+        v-bind:class="classes"
+        v-bind:viewBox="viewBox"
+        ref="svg"
+        v-on:mousemove="onMouseMove"
+        v-on:touchmove="onTouchMove"
+        v-on:mouseup="onMouseUp"
+        v-on:touchend="onTouchEnd"
+        v-on:touchcancel="onTouchCancel"
+        v-on:touchleave="onTouchLeave"
+    >
+        <line
+            class="slope"

```

Comment:
Maybe add a bit more info
```suggestion
            class="slope slope-1"
```